runtime/doc/eval.txt

```diff
@@ -4545,6 +4545,7 @@ getenv({name})						*getenv()*
 		When the variable does not exist |v:null| is returned.  That
 		is different from a variable set to an empty string.
 		See also |expr-env|.
+			getenv('HOME')
```

Comment:
These should be fenced, like:

```suggestion
		See also |expr-env|. Example: <
			getenv('HOME')
```

Bram may have fixed these for you in the merge though. :+1: